launch/VirSat CEF Application (user).launch

```diff
@@ -59,8 +59,6 @@
 <setEntry value="org.eclipse.ecf.core.feature:default"/>
 <setEntry value="org.eclipse.ecf.core.ssl.feature:default"/>
 <setEntry value="org.eclipse.ecf.filetransfer.feature:default"/>
-<setEntry value="org.eclipse.ecf.filetransfer.httpclient4.feature:default"/>

```

Comment:
Should we replace it with httpclient45 or is it not used anywhere?

Reply:
Its in out product configuration, so I think we should start it as well in here.